components/brave_new_tab_ui/stories/default.tsx

```diff
@@ -3,22 +3,52 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import * as React from 'react'
+import { Provider as ReduxProvider } from 'react-redux'
 import { storiesOf } from '@storybook/react'
-import { withKnobs, select, boolean } from '@storybook/addon-knobs/react'
+import { withKnobs, select } from '@storybook/addon-knobs/react'
+import Theme from 'brave-ui/theme/brave-default'
+import DarkTheme from 'brave-ui/theme/brave-dark'
+import BraveCoreThemeProvider from '../../common/BraveCoreThemeProvider'
 
 // Components
-import NewTabPage from './default/index'
+import NewTabPage from '../containers/newTab'
```

Comment:
This is my main issue. I'm ok if this is a one off and we don't have time to do this correctly, but in pure form, a story should not import a Container, only Components. The Container should have the logic for the specific data context it's in, and the Story should have the same logic for the story / debug context it's in. But as mentioned, the way you have the overrides moved to the redux store makes the effect not noticeable for the storybook user / developer, so if the main aim is removal of code duplication then it's good. As long as we also file the issue to separate.

Reply:
My selling point for having this container as story is to have some sort of hot-reload so I could speed-up design tweaks. Not sure what is your suggestion. You mean having only components in stories and not containers?

Reply:
By the way I don't have any personal preference with Storybook, goal is mostly getting rid of duplicates

Reply:
Yeah only having components but that the whole NTP would be a component.

Anyway I think this is great, as long as we only do it for 'full pages'. So IMO feel free to merge when you fix any of the nits you want to.

Reply:
Gotchu